owncloudApp/src/main/java/com/owncloud/android/presentation/files/createfolder/CreateFolderDialogFragment.kt

```diff
@@ -82,12 +87,48 @@ class CreateFolderDialogFragment : DialogFragment() {
         alertDialog.setOnShowListener {
             val okButton = alertDialog.getButton(AlertDialog.BUTTON_POSITIVE)
             okButton.isEnabled = isButtonEnabled
+
+            okButton.setOnClickListener {
+                var fileName: String = inputText.text.toString()
+                if (fileName.isEmpty()) {
+                    error = getString(R.string.uploader_upload_text_dialog_filename_error_empty)
+                } else {
+                    null
+                }
+                if (error != null) {
+                    inputLayout.isErrorEnabled = true
+                    inputLayout.error = error
+                } else {
+                    createFolderListener.onFolderNameSet(fileName, parentFolder)
+                    dialog?.dismiss()
+                }
+            }
         }
 
         inputText.doOnTextChanged { text, _, _, _ ->
             val okButton = alertDialog.getButton(AlertDialog.BUTTON_POSITIVE)
-            isButtonEnabled = !text.isNullOrBlank()
-            okButton.isEnabled = isButtonEnabled
+
+            if (text.isNullOrBlank()) {
+                okButton.isEnabled = false
```

Comment:
I think this behaviour is provided by the InputText by default (or at least it was), so check it and if it's not needed, we can remove this code 😀

Reply:
Yes, this behavior was before the issue in that class. I just readapt the code. It wasn't provided by the InputText.